data/en/test_unit_patterns.json

```diff
@@ -30,35 +30,71 @@
                         "label": "Centimetres",
                         "mandatory": false,
                         "type": "Unit",
-                        "unit": "length-centimeter"
+                        "unit": "length-centimeter",
+                        "length": "short"
                     }, {
                         "id": "metres",
                         "description": "",
                         "label": "Metres",
                         "mandatory": false,
                         "type": "Unit",
-                        "unit": "length-meter"
+                        "unit": "length-meter",
+                        "length": "short"
                     }, {
                         "id": "kilometres",
                         "description": "",
                         "label": "Kilometres",
                         "mandatory": false,
                         "type": "Unit",
-                        "unit": "length-kilometer"
+                        "unit": "length-kilometer",
+                        "length": "short"
                     }, {
                         "id": "miles",
                         "description": "",
                         "label": "Miles",
                         "mandatory": false,
                         "type": "Unit",
-                        "unit": "length-mile"
+                        "unit": "length-mile",
+                        "length": "short"
                     }],
                     "description": "",
                     "id": "set-length-units-question",
                     "title": "Length Units",
                     "type": "General"
                 }],
                 "title": "Length Units"
+            }, {
+                "type": "Question",
+                "id": "set-duration-units-block",
+                "description": "",
+                "questions": [{
+                    "answers": [{
+                            "id": "duration-hour",
+                            "description": "",
+                            "label": "Hour",
+                            "mandatory": false,
+                            "type": "Unit",
+                            "unit": "duration-hour",
+                            "length": "long",
+                            "use_plural_for_label": true
+                        },
+                        {
+                            "id": "duration-year",
+                            "description": "",
+                            "label": "Years",
+                            "mandatory": false,
+                            "type": "Unit",
+                            "unit": "duration-year",
+                            "length": "long",
+                            "use_plural_for_label": true
```

Comment:
Do we have an example where this is set to false?

Reply:
Added this just before I saw this message ^^